src/main/resources/jenkins/branch/PropertyMigration/MonitorImpl/message.jelly

```diff
@@ -65,7 +66,6 @@
                         enctype="multipart/form-data"
                         disabled="disabled"
                         action="${rootURL}/pluginManager/install"
-                        onsubmit="return addCrumb('install-${p.pluginName}');"
```

Comment:
No alternative, or not needed?

Reply:
I'm not confident here.
From PR description:
> Regarding the removal of addCrumb calls. I might be missing the point. I can't find any traces of such method ([trivial org search](https://github.com/search?q=org%3Ajenkinsci+addCrumb&type=code) doesn't fetch any results).
[This part of core](https://github.com/jenkinsci/jenkins/blame/d0c0d07bc4644bc7045ac321472d0a9822dbb8b8/war/src/main/webapp/scripts/hudson-behavior.js#L1450-L1474) has been there for a while, so I'm not sure what that inline extra call is supposed to do. Without it plugin can be installed just fine as you can see on the screen recording.